### ISSUE TITLE
Suggestion: New technologies from the Free Worlds war should be released to be used by other factions

### ISSUE THREAD

**Issue by Bladewood** [endless-sky/endless-sky#1244]
For example, electron beams should eventually be released to the general public by the Deep and sometimes be found on merchant ships. And, the Syndicate's shield regenerators and their ionic afterburners should also eventually be released and found on merchant ships.
Selfish factions always create distrust, which will probably start another war. We don't want war, right?
...well, if we do, at least it makes for a fun game.

**Comment by Alkallid** [comment]
So at the end of each campaign, all tech slowly spreads out? I think it would work, but the Republic would be very wary. And do you mean you want to see a Dreadnought escorting some freighters and such?

**Comment by jafdy** [comment]
I can't see the Dreadnought (and Skein) being sold to the general public. It is easy to see the Syndicate selling their stuff at some point. The plasma turret and electron beams are harder, my feeling is no, but I see no reason why not.

If this goes through some one will need to make the ships/fleets for it.

**Comment by LocalGod79** [comment]
i think, lore wise, that there would be nothing stopping Southbound from selling their Dreadnoughts to non-FW factions - there's nothing to suggest they are restricting it any more than any of the other shipyards are restricting the sale of their Heavy Warships.

the Skein was given a license requirement, granted upon holding a Free Worlds salary - a small "perk" for choosing that arc, in anticipation of the coming arcs which will unlock their own ships.

that said, i believe it to be a good plan to have those technologies be released after the three main arcs are completed, and some more story content is added.

**Comment by Bladewood** [comment]
What does the player completing the other arcs have to do with humanity and them adding new equipment to their outfitters? Pretty sure "arc" refers to "campaign": Free Worlds, Wanderers, and the future third one.
*If I'm wrong, ignore this whole textwall.* But if I'm right, it would be a veeery strange coincidence for humanity's new equipment to be released only after we end the Free Worlds war, bring peace to Wanderer life, and do whatever the third one is supposed to be.

**Comment by Alkallid** [comment]
No, he mant, "Write the damn story first." Also, by three arcs, it was Free Worlds, Syndicate, and Republic, -you orange dunce.-

**Comment by Amazinite** [comment]
He's saying that once the Free Worlds, Syndicate, and Republic campaigns are all done, then we can decide on how the technology becomes available to the "public."

**Comment by jafdy** [comment]
>to start selling Dreadnoughts to certain select members of the Free Worlds

@LocalGod79 So is this the same lore the lets you buy a Cruiser? If the Republic get the Cruiser and the Carrier then why shouldn't the Free Worlds get the Dreadnought and the Skein.

**Comment by Alkallid** [comment]
I don't see your meaning, but Cruisers and Carriers are also only limited to certain officers, with their respective licences.

**Comment by endless-sky** [comment]
If you side with the Syndicate, you get access to the Syndicate tech and nukes. If you fight the Syndicate, you get access to the new Navy tech (electron beams and Typhoon torpedoes) and a cloaking device.

**Comment by jafdy** [comment]
@Alkallid it's a quote from when drednoughts go on sale. The implication is that those two ships (dreadnought and skein) are only sold to militia members. In the same way that cruisers/carriers are only given to some navy officers.

**Comment by Alkallid** [comment]
But what was the "If the Republic get the Cruiser and the Carrier then why shouldn't the Free Worlds get the Dreadnought and the Skein."?

**Comment by LocalGod79** [comment]
i think one has to realize that the Republic arc is probably going to have options as to which ships get unlocked - there's also the Frigate, Gunboat, and Rainmaker beyond the Cruiser and Carrier - and who-knows-what-else...

like i said - the Skein was given the FW arc restriction so that the arc wouldn't be *totally* devoid of locked ships, and lore wise: a carrier with 6 fighters is a significant force; one that the FW would naturally rather avoid selling to Southern Pirates.

i always presumed the dialogue;
> Freya just contacted me, to say that they're now ready to start selling Dreadnoughts to certain select members of the Free Worlds, including you.

implied that production had finally started, and the first several ships were reserved for sale to the Militia, rather than private captains.

**Comment by Amazinite** [comment]
>If you side with the Syndicate, you get access to the Syndicate tech and nukes. If you fight the Syndicate, you get access to the new Navy tech (electron beams and Typhoon torpedoes) and a cloaking device.

@endless-sky [The end of FW Checkmate](https://github.com/endless-sky/endless-sky/blob/master/data/free%20worlds%20checkmate.txt#L2647) provides both Syndicate tech and Deep tech. Is this not suppose to be the case, because the way you worded that sounds like it shouldn't be.

**Comment by jafdy** [comment]
Checkmate needs all the benefits it can get, people prefer playing Rec and prefer the reward at the end.

**Comment by KiLEdEnNis** [comment]
am I the only one that chose Checkmate 1st time ever in his like 4th playthrough & then never did Reconcilation again?

**Comment by Bladewood** [comment]
Yes.
